src/sentry/static/sentry/app/routes.jsx

```diff
@@ -789,6 +789,13 @@ function routes() {
             />
           </Route>
 
+          <Route
+            path="/organizations/:orgId/releases/"
```

Comment:
I'm a bit confused about why we'd need to add to `RESERVED_PROJECT_SLUGS`. Since the path is prefixed with `/organizations/:orgId/` I think there isn't a possibility of a naming collision?

Reply:
Oops my mistake.